index.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset="UTF-8" />
+    <title>Hello React Typescript!</title>
+</head>
+
+<body>
+    <div id="demo-app"></div>
+
+    <!-- Dependencies -->
+    <script src="./node_modules/react/umd/react.development.js"></script>
+    <script src="./node_modules/react-dom/umd/react-dom.development.js"></script>
+
+    <!-- Main -->
+    <script src="./dist/demo-app.js"></script>
+</body>
+
+</html>
```

Comment:
New line should be at the end of every file. This is one of the linting rule and we are following it.

Reply:
Okay !! Will do it !!